### PR TITLE
Language: Make production error message translatable

### PR DESCRIPTION
**Pull request by sba**
With this pull request the error message in the file `app/Views/errors/html/production.php` can be translated.

I have named the file Errors.php because the translation of `app/Views/errors/html/error_exception.php` could also be added later.

### CODE REVIEW

**Comment by kenjis** [comment]
Thank you for sending a PR.

You must [GPG-sign](https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/signing.md) your work, certifying that you either wrote the work or otherwise have the right to pass it on to an open-source project. See [Developer's Certificate of Origin](https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/DCO.md).
See https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/pull_request.md#signing

File: app/Views/errors/html/production.php [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8">
     <meta name="robots" content="noindex">
 
-    <title>Whoops!</title>
+    <title><?= lang('Errors.whoops')?></title>
```
Comment (kenjis) [inline app/Views/errors/html/production.php:7]:
```suggestion
    <title><?= lang('Errors.whoops') ?></title>
```

File: app/Views/errors/html/production.php [hunk 2]
```diff
@@ -14,9 +14,9 @@
 
     <div class="container text-center">
 
-        <h1 class="headline">Whoops!</h1>
+        <h1 class="headline"><?= lang('Errors.whoops')?></h1>
```
Comment (kenjis) [inline app/Views/errors/html/production.php:17]:
```suggestion
        <h1 class="headline"><?= lang('Errors.whoops') ?></h1>
```

File: app/Views/errors/html/production.php [hunk 3]
```diff
@@ -14,9 +14,9 @@
...
 
-        <p class="lead">We seem to have hit a snag. Please try again later...</p>
+        <p class="lead"><?= lang('Errors.weHitASnag')?></p>
```
Comment (kenjis) [inline app/Views/errors/html/production.php:19]:
```suggestion
        <p class="lead"><?= lang('Errors.weHitASnag') ?></p>
```

**Comment by sba** [comment]
Hi @kenjis 
Thanks for checking my PR and the GPG links. I will try to GPG-sign my commit...never did this before. Should I close this PR and make a new, signed one?
And is the naming "Errors.php" acceptable or do you have a better suggestion?
I thought about another name in case welcome_message.php should be made translatable later - but I think it would be better to name it for this Welcome.php anyway.

**Comment by kenjis** [comment]
> Should I close this PR and make a new, signed one?

No. You can do `git commit --amend`, and force push.

**Comment by kenjis** [comment]
Agreed. It seems better to go to 4.3.

**Comment by sba** [comment]
> 2. The new language file will require adoption by the translations repo before this will actually work.

I will have a look to the translations repo and try to update the necessary things (tests etc.)

**Comment by MGatner** [comment]
Thanks @sba. To clarify, this should work fine without the translations (it falls back to English), but any additions you can make to the translations repo to go along with it would be appreciated!

**Comment by ddevsr** [comment]
With this PR, must include in :
https://github.com/codeigniter4/CodeIgniter4/blob/develop/app/Views/errors/html/error_404.php#L79

**Sorry! Cannot seem to find the page you were looking for.** can multilingual too

**Review by kenjis — CHANGES_REQUESTED** [review]
This should go to 4.3 branch. Please change the base branch.

**Review by kenjis — APPROVED** [review]
LGTM!

**Comment by kenjis** [comment]
@sba You forgot to sign your commits.
See https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/pull_request.md#signing

**Comment by sba** [comment]
> @sba You forgot to sign your commits. See https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/pull_request.md#signing

Yes, I know, sorry! It's a crap, I've wasted hours on it and signing still doesn't work...... I'll try again, sometime.

**Review by MGatner — APPROVED** [review]
Looks good! I like @ddevsr's suggestion to include the 404 error as well, but that could be a separate PR if you're done with this.

**Review by kenjis — CHANGES_REQUESTED** [review]
There are many unrelated commits.
Please drop these.

**Comment by kenjis** [comment]
@sba Thank you for GPG signing.

But unfortunately this PR branch has many unrelated commits.
You need to clean up the branch.

I recommend you create a new branch from `4.3`, 
See https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/workflow.md#branching-revisited
and cherry-pick commits you made.
After that remove this  PR branch `develop`, and rename the new branch to `develop` and force push.

Or it is easier just to create another PR.

**Pull request closed without merging**